examples/platform/silabs/efr32/wf200/efr_spi.c

```diff
@@ -316,55 +287,24 @@ sl_status_t sl_wfx_host_spi_transfer_no_cs_assert(sl_wfx_host_bus_transfer_type_
     return SL_STATUS_OK;
 }
 
-/****************************************************************************
- * @fn   void sl_wfx_host_start_platform_interrupt(void)
- * @brief
- * Enable WFX interrupt
- * @param[in]  none
- * @return None
- *****************************************************************************/
 void sl_wfx_host_start_platform_interrupt(void)
 {
     // Enable (and clear) the bus interrupt
     GPIO_ExtIntConfig(SL_WFX_HOST_PINOUT_SPI_WIRQ_PORT, SL_WFX_HOST_PINOUT_SPI_WIRQ_PIN, wirq_irq_nb, true, false, true);
 }
 
-/****************************************************************************
- * @fn   sl_status_t sl_wfx_host_disable_platform_interrupt(void)
- * @brief
- * Disable WFX interrupt
- * @param[in]  None
- * @return  returns SL_STATUS_OK if successful,
- *          SL_STATUS_FAIL otherwise
- *****************************************************************************/
 sl_status_t sl_wfx_host_disable_platform_interrupt(void)
 {
     GPIO_IntDisable(1 << wirq_irq_nb);
     return SL_STATUS_OK;
 }
 
-/****************************************************************************
```

Comment:
Now why removed function headers ? Don't we want the documentation ?

Reply:
> the documentation has been moved to the common header in `spi_multiplex.h`